java-checks/src/main/resources/org/sonar/l10n/java/rules/java/S6395_java.html

```diff
@@ -0,0 +1,20 @@
+<p>Sub-patterns can be wrapped by parentheses to build a group. This enables to restrict alternations, back reference the group or apply quantifier to
+the sub-pattern.</p>
+<p>If this group should not be part of the match result or if no reference to this group is required, a non-capturing group can be created by adding
+<code>:?</code> behind the opening parenthesis.</p>
```

Comment:
Oops, it's `<code>?:</code>`

Reply:
Change done: https://github.com/SonarSource/rspec/pull/982